packages/dds/map/src/interfaces.ts

```diff
@@ -144,16 +134,17 @@ export interface IDirectoryValueChanged extends IValueChanged {
 }
 
 export interface ISharedMapEvents extends ISharedObjectEvents {
-    (event: "valueChanged", listener: (
-        changed: IDirectoryValueChanged,
+    valueChanged(
+        changed: IValueChanged,
```

Comment:
Note that this was wrong/mismatched before. In the emit calls, it doesn't send an `IDirectoryValueChanged`, but rather an `IValueChanged`. We could change the emit calls or the typing. Not sure, but they should be synced.

Reply:
Because I did it this way, I needed to change a bunch of downstream code to dependon `IValueChanged` instead of `IDirectoryValueChanged`. Curious what is correct here? - for example the react package.

Reply:
Oh sorry, I just fixed this today: https://github.com/microsoft/FluidFramework/pull/6397